samples/protected-setting-sample/README.md

```diff
@@ -0,0 +1,45 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Protected settings sample
+
+This sample shows how ISV can create protected settings and input secret values in UIDefinition as well as consume them in ARM Template so that they can be passed to Kubernetes Extension Type as Configuration Protected Settings.
+
+Protected configuration settings help manage sensitive data securely. These settings typically include information such as passwords, API keys, certificates, or connection strings. 
+- Protected configuration settings are stored as secrets on the Kubernetes clusters. As a result, the key names of the protected configuration settings should be simple text and special characters should not be included. 
```

Comment:
Please add below text at end of this statement
For naming constraints, refer [here.](https://kubernetes.io/docs/concepts/configuration/secret/#restriction-names-data)